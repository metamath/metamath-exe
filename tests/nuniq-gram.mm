$( Here we have two variable typecodes A and B, and one logical
   typecode |-, and the question is whether |- connects to A or B.
   It parses either way. This is actually an unambiguous formal system,
   because we require only that there are no two ways to derive the proof
   of the *same* syntax theorem, and "A *" and "B *" are different theorems. $)
$c A B |- * $. $v a b $.
va $f A a $.
vb $f B b $.
as $a A * $.
bs $a B * $.
ax $a |- * $.
