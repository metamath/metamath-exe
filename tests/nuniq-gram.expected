MM> READ "nuniq-gram.mm"
Reading source file "nuniq-gram.mm"... 448 bytes
448 bytes were read into the source buffer.
The source has 7 statements; 3 are $a and 0 are $p.
No errors were found.  However, proofs were not checked.  Type VERIFY PROOF *
if you want to check them.
MM> Continuous scrolling is now in effect.
MM> 0 10%  20%  30%  40%  50%  60%  70%  80%  90% 100%
..................................................
All proofs in the database were verified.
